### PR TITLE
fix(home sections): Fix no items appearing in home sections

### PR DESCRIPTION
**Pull request by Camc314**
This fixes no homesections appearing

This does not yet fix the latest items home section

### CODE REVIEW

**Comment by Camc314** [comment]
I'm having an issue with computed properties whereby:

1.  loading the home screen initial  - latest media does not appear.
2. Clicking on a library and returning makes the items appear

I'm not entirely sure what the best method is to fix this

File: components/HomeSection.vue [hunk 1]
```diff
@@ -72,7 +73,26 @@ export default Vue.extend({
     };
   },
   computed: mapState<AppState>({
-    items: (state: AppState) => state.homeSection.items
+    items(state: AppState): BaseItemDto[] {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```
Comment (ferferga) [inline components/HomeSection.vue:78]: Why?

Comment (Camc314) [reply components/HomeSection.vue:78]: I was getting `any Property 'section' does not exist on type '{}'.` errors. Not entirely sure why 🤷‍♀️

Comment (ferferga) [reply components/HomeSection.vue]: @Camc314 Fixed them, hope you don't mind!

**Review by ferferga — APPROVED** [review]

**Comment by Camc314** [comment]
**Don't merge yet**

Latest media sections are still broken

**Review by MrTimscampi — APPROVED** [review]

**Pull request merged**